ui/next/Makefile

```diff
@@ -33,7 +33,7 @@ all: lint $(REMOTE_DEPS)
 
 .PHONY: lint
 lint: npm.installed
-	tslint -c $(TS_ROOT)/tslint.json $(shell find $(TS_ROOT) -name '*.tsx' -or -name '*.ts' -not -path app/interfaces/protos.d.ts)
+	tslint -c tslint.json $(shell find $(TS_ROOT) -name '*.tsx' -or -name '*.ts' -not -path app/interfaces/protos.d.ts)
```

Comment:
done

On Wed, May 4, 2016 at 12:46 PM, mrtracy <notifications@github.com> wrote:

> In ui/next/Makefile
> <https://github.com/cockroachdb/cockroach/pull/6485#discussion_r62073294>:
>
> > @@ -33,7 +33,7 @@ all: lint $(REMOTE_DEPS)
> >
> >  .PHONY: lint
> >  lint: npm.installed
> > -	tslint -c $(TS_ROOT)/tslint.json $(shell find $(TS_ROOT) -name '*.tsx' -or -name '*.ts' -not -path app/interfaces/protos.d.ts)
> > +	tslint -c tslint.json $(shell find $(TS_ROOT) -name '*.tsx' -or -name '*.ts' -not -path app/interfaces/protos.d.ts)
>
> Don't forget to change the build targets from lib/protos to app/js
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/6485/files/c0265f636df54be0cefe98a20a8a6a4317ebc12a#r62073294>
>